packages/mutation-testing-elements/test/unit/components/mutation-test-report-file-legend.spec.ts

```diff
@@ -34,16 +35,40 @@ describe(MutationTestReportFileLegendComponent.name, () => {
     await sut.updateComplete;
     const actualCheckboxes = sut.$$('.form-check.form-check-inline');
     expect(actualCheckboxes).lengthOf(6);
-    const checkboxTexts = actualCheckboxes.map(checkbox => (checkbox.textContent as string).trim());
-    expect(checkboxTexts).deep.eq(['Killed (1)',
-      'Survived (1)',
-      'NoCoverage (1)',
-      'Timeout (1)',
-      'CompileError (1)',
-      'RuntimeError (1)'
+    const checkboxTexts = actualCheckboxes.map(checkbox => normalizeWhitespace((checkbox.textContent as string)));
+    expect(checkboxTexts).deep.eq([
+      '✅ Killed (1)',
+      '👽 Survived (1)',
+      '🙈 NoCoverage (1)',
+      '⌛ Timeout (1)',
+      '💥 CompileError (1)',
+      '💥 RuntimeError (1)'
     ]);
   });
 
+  it('should show the correct colors for the states', async () => {
```

Comment:
No idea if this is possible in JS, but wouldn't it be better to do something like:
```js
expectedMutantColors.forEach((status, expectedColor) => {
  it(`should show the color ${expectedColor} for status ${status}`, () => ...);
});
```

Edit: great minds think alike? You already have that in a test below. So why not here?

Reply:
With "below", you mean in another file 😌

No idea why I guess I wrote them on separate days? Fixing it now...